infrastructure/docker-compose.deploy.yml

```diff
@@ -15,12 +15,8 @@ services:
   traefik:
     image: 'traefik:v2.9'
     ports:
-      - target: 80
-        published: 80
-        mode: host
-      - target: 443
-        published: 443
-        mode: host
```

Comment:
This was ensuring that the client's IP gets passed through traefik. More on this here: https://www.reddit.com/r/Traefik/comments/x48zvv/forwarding_real_ip_when_running_as_a_container/

Reply:
then again if the client is behind a VPN then does that even make sense to track their IP address?

Reply:
Good catch.. In that case, you are right, there's really no IP address to track. 
So currently the requirements for this setup would be:
- network mode needs to be `host` so if VPN is not use, we track the right IPs
- access should only be allowed from `VPN_HOST_ADDRESS` if it's set

I did some digging on this and it seems you can only achieve the first requirement with the object syntax and you can only do the second requirement with the string syntax 🤦‍♂️ 

Ideally, any IP limiting should be done on a network level above the application server. Alternative is to make Traefik block IPs with 403 but in my opinion, that's pushing the limitation to the wrong direction, deeper into our stack. Non VPN clients could also then know that there is something running on the web ports and could use this for malicious purposes.

I'm gonna just revert this change for now.